servers/register_server_types.cpp

```diff
@@ -60,14 +60,7 @@
 #include "movie_writer/movie_writer.h"
 #include "movie_writer/movie_writer_mjpeg.h"
 #include "movie_writer/movie_writer_pngwav.h"
-#include "navigation_server_2d.h"
 #include "navigation_server_3d.h"
```

Comment:
I would also move this in the "3D physics + navigation" block, and just keep it outside the `#ifndef _3D_DISABLED` (with a comment justifying it's needed for 2D).